about/about-kubernetes-networking.md

```diff
@@ -0,0 +1,79 @@
+
+### The Kubernetes network model
+
+The Kubernetes network model specifies:
+- Every pod gets its own IP address
+- Containers within a pod share the pod IP address and can communicate freely with each other
+- Pods can communicate with all other pods in the cluster using pod IP addresses (without [NAT]({{site.baseurl}}/about/about-networking#nat))
+- Isolation (restricting what each pod can communicate with) is defined using network policies
+
+As a result, pods can be treated much like VMs or hosts (they all have unique IP addresses), and the containers within pods very much like processes running within a VM or host (they run in the same network namespace and share an IP address). This model makes it easier for applications to be migrated from VMs and hosts to pods managed by Kubernetes. In addition, because isolation is defined using network policies rather than the structure of the network, the network remains simple to understand. This style of network is sometimes referred to as a "flat network".
+
+Note that, although very rarely needed, Kubernetes does also support the ability to map host ports through to pods, or to run pods directly within the host network namespace sharing the host's IP address.
+
+### Kubernetes network implementations
+
+Kubernetes built in network support, kubenet, can provide some basic network connectivity. However, it is more common to use third party network implementations which plug into Kubernetes using the CNI (Container Network Interface) API.
+
+There are two basic kinds of CNI plugins:
+- network plugins, which are responsible for connecting pod to the network
+- IPAM (IP Address Management) plugins, which are responsible for allocating pod IP addresses.
+
+{{site.prodname}} provides both network and IPAM plugins, but can also integrate and work seamlessly with some other CNI plugins, including AWS, Azure, and Google network CNI plugins, and the host local IPAM plugin. This flexibility allows you to choose the best networking options for your specific needs and deployment environment. You can read more about this in the {{site.prodname}} [determine best networking option]({{site.baseurl}}/networking/determine-best-networking) guide.
+
+### Kubernetes Services
+
+Kubernetes Services provide a way of abstracting access to a group of pods as a network service. The group of pods is usually defined using a {% include open-new-window.html text='label selector' url='https://kubernetes.io/docs/concepts/overview/working-with-objects/labels' %}. Within the cluster the network service is usually represented as virtual IP address, and kube-proxy load balances connections to the virtual IP across the group of pods backing the service. The virtual IP is discoverable through Kubernetes DNS.  The DNS name and virtual IP address remain constant for the life time of the service, even though the pods backing the service may be created or destroyed, and the number of pods backing the service may change over time.
```

Comment:
I wouldn't say "network service" here, just "service".

It might be simpler to just say that the service usually has an IP address and DNS name that kube-proxy load balances...

Reply:
I wanted to use the same terminology as the k8s docs.  And they say "network service".